### PR TITLE
Mp profile views

### PR DESCRIPTION
**Pull request by pattmax00**
Everything here is done, styling needs to be improved

### CODE REVIEW

File: AcademicReward/Views/GroupsPage.xaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="AcademicReward.Views.GroupsPage"
+             Title="Your Groups">
+
+    <ContentPage.Resources>
+        <Style TargetType="Button">
```
Comment (Wil-LaLonde) [inline AcademicReward/Views/GroupsPage.xaml:8]: Does this apply to all buttons on the page then? If so, I'm stealing this

File: AcademicReward/Views/GroupPage.xaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            FontSize="Large"
+            Text="Example Group Name"
+            VerticalOptions="Center"
+            HorizontalOptions="Center" />
+        <Label
+            FontSize="Medium"
+            Text="This is where the description will go"
+            VerticalOptions="Center"
+            HorizontalOptions="Center"
+            Margin="20,20,20,20"/>
```
Comment (Wil-LaLonde) [inline AcademicReward/Views/GroupPage.xaml:17]: Not going to make you change this, but this can be simplified to Margin="20"

**Review by Wil-LaLonde — APPROVED** [review]
Looks good, nice work. We'll work on the styling 👍

**Comment by Wil-LaLonde** [comment]
oh, didn't mean to make the comments block merging, will resolve those now

File: AcademicReward/Views/GroupsPage.xaml (hunk 1 above)
Comment (pattmax00) [reply AcademicReward/Views/GroupsPage.xaml:8]: Yes! That's kinda what got me thinking about styling again on Discord. I hope there's a way to do this project wide (I'm sure there is) or at least a way to import a massive style sheet on every page.

**Pull request merged**